person.rb

```diff
@@ -29,4 +29,8 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [OPTIONAL] The `add_rental` method should be added to the person class like so:
```def add_rental(book, date)```
   ``` Rental.new(date, book, self)```
 ``` end```
Please kindly fix that.

Reply:
Alright noted, thanks for that!